setup.py

```diff
@@ -0,0 +1,43 @@
+try:
+    from setuptools import setup
+except:
+    from distutils.core import setup
+import re
+
+__author__ = "Achilleas Koutsou"
+
+install_requires = ["numpy>=1.3.0",
+                    "quantities>=0.9.0"]
```

Comment:
1. `install_requires` is not used in `setup` :-)
2. Those are already dependencies of neo, there is no need to redefine them